test/CMakeLists.txt

```diff
@@ -138,7 +154,7 @@ function(add_performance_test TEST_NAME COMM RMW_IMPLEMENTATION)
 
```

Comment:
Is this still the case:
```
# TODO (ahcorde): perf_test is not working with CycloneDDS or FastRTPS when
# processes are splitted in two
```

Reply:
Fixed comments https://github.com/ros2/buildfarm_perf_tests/pull/26/commits/335939cb32228f2f2fd80776559f887e398ea543

Reply:
:+1: